mono/metadata/gc.c

```diff
@@ -726,10 +726,18 @@ finalize_domain_objects (DomainFinalizationReq *req)
 #elif defined(HAVE_SGEN_GC)
 	while (!suspend_finalizers && (count = mono_gc_finalizers_for_domain (domain, to_finalize, NUM_FOBJECTS))) {
```

Comment:
If it's going to be done, just do it as part of this PR. It's a nice unit of change.